lib/reply.js

```diff
@@ -654,6 +654,13 @@ function notFound (reply) {
   }
 }
 
+/**
+ *
```

Comment:
These are a good start. However, a brief description of the function's purpose is also useful.

Reply:
Sorry I thought I would be redundant on the `@return`, but I should have fixed now 👍